examples/event_handling/pong_sgskip.py

```diff
@@ -319,7 +318,7 @@ def start_anim(event):
 
     def local_draw():
         if animation.ax.get_renderer_cache():
-            animation.draw(None)
+            animation.draw()
     start_anim.timer.add_callback(local_draw)
```

Comment:
You could just connect animation.draw here and get rid of local_draw.  Note that checking for the presence of a cached renderer is unnecessary because start_anim is triggered by a draw_event below, which occurs *after* the first actual draw occurred (as noted in #17168, for example).